Sources/NIO/Bootstrap.swift

```diff
@@ -967,3 +978,54 @@ public final class NIOPipeBootstrap {
         }
     }
 }
+
+/// An channel option which can be applied to bootstrap using shorthand notation.
+/// - See: ServerBootstrap.serverOptions(_ options: [ShorthandBootstrapOption])
+public struct ShorthandBootstrapOption {
+    private let apply: ShorthandApply
+    
+    private init(apply: ShorthandApply) {
+        self.apply = apply
+    }
+    
+    /// Apply the contained option to the supplied ServerBootstrap
+    /// - Parameter serverBootstrap: bootstrap to apply this option to.
+    /// - Returns: the modified bootstrap (currently the same one mutated)
+    func applyOption<BootstrapT : ServerBootstrap>(serverBootstrap : BootstrapT) -> BootstrapT {
+        return apply.applyOption(serverBootstrap: serverBootstrap)
+    }
+}
+
+private protocol ShorthandApply {
+    func applyOption<BootstrapT : ServerBootstrap>(serverBootstrap : BootstrapT) -> BootstrapT
+}
+
+private struct ShorthandApplyImpl<Option : ChannelOption> : ShorthandApply {
+    let option : Option
+    let value : Option.Value
+    
+    func applyOption<BootstrapT : ServerBootstrap>(serverBootstrap: BootstrapT) -> BootstrapT {
+        return serverBootstrap.serverChannelOption(option, value: value)
+    }
+}
+
+/// Approved shorthand options.
+extension ShorthandBootstrapOption {
+    /// Option to reuse address.
+    /// - See:  NIOBSDSocket.Option.reuseaddr
+    public static let reuseAddr = ShorthandBootstrapOption(apply: ShorthandApplyImpl(option: ChannelOptions.socketOption(.reuseaddr), value: 1))
+    
+    /// Option to disble autoRead
+    /// - See: ChannelOptions.autoRead
+    public static let disableAutoRead = ShorthandBootstrapOption(apply: ShorthandApplyImpl(option: ChannelOptions.autoRead, value: false))
+    
+    /// `BacklogOption` allows users to configure the `backlog` value as specified in `man 2 listen`. This is only useful for `ServerSocketChannel`s.
+    /// - See: ChannelOptions.backlog
+    public static func backlog(_ value : ChannelOptions.Types.BacklogOption.Value) -> ShorthandBootstrapOption {
+        return makeSBO(option: ChannelOptions.backlog, value: value)
```

Comment:
this uses an implicit `Self.` (which doesn't work in 5.0), so we should prefix it with `ServerBootstrap.` which works everywhere.

Reply:
This function died in a later revision.